### PR TITLE
Mr type tester

### PR DESCRIPTION
**Pull request by Maggieryder**
Issue Type tester #1

type tester component from DL ported over to carbon

Todo: insert the correct font-family classes

### CODE REVIEW

File: packages/addons-website/scss/components/_type-tester.scss [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    .#{$prefix}--input-range-wrapper {
+      width: 25%;
+    }
+  }
+}
+
+.#{$prefix}--type-tester-sample {
+  width: 100%;
+  background: $ui-02;
+  // min-height: rem(240px);
```
Comment (alisonjoseph) [inline packages/addons-website/scss/components/_type-tester.scss]: can we remove this line?

**Comment by alisonjoseph** [comment]
Arabic was just added in v`1.3.0` of Plex, can we add here? Also, can we document how to add a new font.

**Comment by Maggieryder** [comment]
I've added a README.md file describing how t add a new language. 
I can add Arabic if you can send me the class name and some display text

**Comment by alisonjoseph** [comment]
@Maggieryder where are you loading the fonts from? I'm not sure where the class name is coming from for the other fonts.

**Comment by Maggieryder** [comment]
@alisonjoseph 
This is what is imported at the top of src/styles/index.scss
'@carbon/type/scss/type';
'@carbon/type/scss/font-face/sans';
'@carbon/type/scss/font-face/serif';
'@carbon/type/scss/font-face/mono';
'@carbon/type/scss/classes';

**Comment by alisonjoseph** [comment]
oh ok, we still need to get it added to the type package then first

**Comment by Maggieryder** [comment]
@alisonjoseph 
Though I'm not clear on how the Greek, Hebrew, Thai fonts are getting in yet...

**Comment by Maggieryder** [comment]
Just spoke to Ryan. Apparently the fonts on the site are coming from a CDN, with the exception of the non-latin language fonts. He is in the process of coming up with a plan to incorporate these fonts into the carbon-website.

**Comment by Maggieryder** [comment]
@alisonjoseph I've been trying to see why this deploy failed but can only see that it timed out on building the site. The storybook completed successfully. Any ideas? Or instructions to retry the checks process

**Comment by alisonjoseph** [comment]
I think it is just taking too long for Netlify, if the circleci build is passing we are ok. I'll retry the Netlify build.

**Comment by vpicone** [comment]
I wonder if it could have anything to do with the size of plex? If so, This PR should help significantly @alisonjoseph https://github.com/IBM/plex/pull/240

**Comment by alisonjoseph** [comment]
this is the error

```
error An unexpected error occurred: "/home/circleci/repo/packages/addons-website/package.json: Unexpected string in JSON at position 1113".
info If you think this is a bug, please open a bug report with the information provided in "/home/circleci/repo/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
Exited with code 1
```

**Comment by Maggieryder** [comment]
@alisonjoseph I have resolved conflicts my end. Please try again

**Comment by Maggieryder** [comment]
@vpicone I have removed all unused fonts in my last commit which should cut down the file size.

**Comment by alisonjoseph** [comment]
closing in favor of #1393

**Pull request closed without merging**